source/includes/_file_proxy_api.md.erb

```diff
@@ -0,0 +1,78 @@
+# HMDA File Serving
+
+The HMDA Platform serves a number of files including:
+
+1. Documentation
+1. Modified LAR
+1. Institution Register Summaries
+1. Dynamic National Loan-Level Dataset
+1. Snapshot National Loan-Level Dataset
+1. Rate Spread Data
+
+The HMDA Platform serves these files in two seperate ways, through the HMDA File api or direct S3 links.
+
+## HMDA File API
+
+The HMDA File API is used for accessing the Modified LAR and Institution Register Summaries.
+
+### Get Modified LAR Pipe Delimited .txt File
+
+ | |
+---|---
+Method | `GET`
+Endpoint | `<%= config[:fileapi] %>/modifiedLar/year/{{year}}/institution/{{lei}}/txt`
+
+### Get Modified LAR Pipe Delimited .txt File with Header
+
+ | |
+---|---
+Method | `GET`
+Endpoint | `<%= config[:fileapi] %>/modifiedLar/year/{{year}}/institution/{{lei}}/txt/header`
+
+### Get Modified LAR .csv File
+
+ | |
+---|---
+Method | `GET`
+Endpoint | `<%= config[:fileapi] %>/modifiedLar/year/{{year}}/institution/{{lei}}/csv`
+
+### Get Modified LAR .csv File with Header
+
+ | |
+---|---
+Method | `GET`
+Endpoint | `<%= config[:fileapi] %>/modifiedLar/year/{{year}}/institution/{{lei}}/csv/header`
+
+### Get Institution Register Summary (IRS)
+
+> **Example Request and Response**
+
+<pre><code class="text">
+<b>Request:</b>
+
+curl -X POST "<%= config[:ratespreadapi] %>/reports/irs/year/2019/institution/B90YWS6AFX2LGWOXJ1LD" \
+-H 'Authorization: Bearer  {{access_token}}'
+
+<b>CSV Response:</b>
+MSA/MD, MSA/MD Name, Total Lars, Total Amount ($000's), CONV, FHA, VA, FSA, Site Built, Manufactured, 1-4 units, 5+ units, Home Purchase, Home Improvement, Refinancing, Cash-out Refinancing, Other Purpose, Purpose N/A
+16984,"Chicago-Naperville, IL-IN-WI", 1, 135, 0, 1, 0, 0, 1, 0, 1, 0, 0, 1, 0, 0, 0, 0
+27260,"Jacksonville-St. Marys-Palatka, FL-GA", 1, 445, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 0
+</code></pre>
+
+The IRS is a summary of a institution's HMDA submission to be used by the filing institution to confirm that they are submitting the correct data.
+
+As such, access to each IRS is restricted to users associated with the institution. Users must be authenticated using the HMDA authentication system and include a bearer token with their request.
+
+(More information on authentication can be found above in the [Authorization section](https://cfpb.github.io/hmda-platform/#hmda-platform-filing-api-authorization) in the HMDA Platform Filing API section of these docs)
+
+ | |
+---|---
+Method | `GET`
+Endpoint | `<%= config[:fileapi] %>/reports/irs/year/{{year}}/institution/{{lei}}`
+Headers | `Authorization: Bearer {{access_token}}`
+
+## Direct S3 File Service
+
+All other files are accessed through direct links to the HMDA Platform's public S3 bucket.
+
+These S3 links are all prefixed with: `<%= config[:s3api] %>`. The list of available files can be found at [ffiec.cfpb.gov](ffiec.cfpb.gov).
```

Comment:
Should we link users back to the list above since there's no *single* list for all files?